samples/esmulti/README.md

```diff
@@ -6,22 +6,24 @@ client/ingest. The sample deploys the nodes as primary (master) nodes and update
 Although multi-role Elasticsearch nodes are widely used, they are not natively supported by the Open Distro for Elasticsearch
 Helm chart that is used to deploy the logging components of this project. In order to enable multi-role Elasticsearch nodes, the deployment process must patch various Kubernetes objects. Because performance and other criteria are still being evaluated, this use of multi-role nodes is experimental and is not yet part of our standard configuration.
 
-## Preparation and Deployment
+## Using This Sample
 
-1. Copy the `../samples/esmulti` sample directory (including sub-directories) to a separate local path.
+You customize your logging deployment by specifying values in `user.env` and `*.yaml` files. These files are stored in a local directory outside of your repository that is identified by the `USER_DIR` environment variable. The configuration files in this sample provide a starting point for the configuration files for a deployment that supports multi-role Elasticsearch nodes. See the 
+[main README](../../README.md#customization) to for information about the customization process.
 
-See the [main README](../../README.md#customization) for information about the customization process.
+If you also need to use values from another sample, manually copy the values to your configuration files after you add the values in this sample. 
 
-2. Set the `USER_DIR` environment variable to the local path:
+After you finish modifying the configuration files, deploy logging using the standard deployment script:
 
 ```bash
-export USER_DIR=/path/to/my/copy/esmulti
+/my_repository_path/logging/bin/deploy_logging_open.sh
 ```
```

Comment:
Shouldn't we still show setting the USER_DIR env var?

Reply:
None of the other samples do - setting USER_DIR is covered in the main README